cli/src/cli/builder/TestBuilder.mjs

```
@@ -0,0 +1,1406 @@
+            break;
+          case 'addTest':
+            setPhase('addTest');
+            break;
+          case 'debugTest':
+            // If only one test with steps, go directly to debug
+            const runnableTests = tests.filter((t) => (t.steps || []).length > 0);
+            if (runnableTests.length === 1) {
+              const runnableIndex = tests.findIndex((t) => (t.steps || []).length > 0);
+              setDebugTestIndex(runnableIndex);
+              setPhase('debugRun');
+            } else {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)